db/migrate/20180606101810_change_file_download_stats_date_to_array.rb

```diff
@@ -0,0 +1,6 @@
+class ChangeFileDownloadStatsDateToArray < ActiveRecord::Migration[5.1]
+  def change
+    remove_column :file_download_stats, :date, :datetime
+    add_column :file_download_stats, :date, :datetime, array: true, default: []
```

Comment:
I think this change may end up breaking this code: https://github.com/samvera/hyrax/blob/be223ff41debc86bac9a5e74ac4151102513e9ac/app/models/hyrax/statistic.rb#L53

Are you sure you don't want to just make a new database table rather than trying to reuse one?